src/sections/services.html

```diff
@@ -0,0 +1,33 @@
+<div
+  class="flex flex-col items-center gap-24 lg:gap-0 lg:flex-row justify-around mt-12 px-8 sm:px-28 lg:px-52"
+>
+  <ul
+    class="lg:text-5xl xl:text-6xl text-black font-[500] leading-[100%] lg:w-large lg:h-large flex flex-col lg:gap-24 text-3xl gap-12"
```

Comment:
Use predefine classes instead of arbitrary values where possilble.
```suggestion
    class="lg:text-5xl xl:text-6xl text-black font-medium leading-[100%] lg:w-large lg:h-large flex flex-col lg:gap-24 text-3xl gap-12"
```